src/datasources/staking-api/staking-api.manager.ts

```diff
@@ -35,11 +41,17 @@ export class StakingApiManager implements IStakingApiManager {
       .getChain(chainId)
       .then(ChainSchema.parse);
 
+    // TODO: remove this check and the associated configuration once Base is
+    // fully migrated to the Kiln mainnet API.
+    const isBaseTestnet =
+      chainId === this.BASE_CHAIN_ID && !this.isBaseProductionActive;
+    const env = isBaseTestnet || chain.isTestnet ? 'testnet' : 'mainnet';
```

Comment:
This targets 'testnet' for Base, regardless of the `chain.isTestnet` flag value. Once the Base contracts are available on Kiln's `mainnet`, the `STAKING_BASE_PRODUCTION_ACTIVE` flag can be used to point `mainnet`.

Once this is done, the flag can also be deleted, as the comments say.